cve_bin_tool/checkers/polarssl_fedora.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for polarssl
+
+https://www.cvedetails.com/product/22470/Polarssl-Polarssl.html?vendor_id=12001
+
+"""
+from . import Checker
+
+
+class PolarsslFedoraChecker(Checker):
```

Comment:
I think the class name is pretty self-evident, but could you put a docstring in here saying again that this only works on fedora (hopefully clear from the class name) and what other versions you tried but couldn't find strings for (not currently recorded outside of this PR, I think?)

I want to err on the side of being clear and leave some hints in case someone wants to work on expanding this to other distros in the future.

Reply:
I have added a comment specifying that it currently only works on fedora, also with the fact that unsuccessful attempts have been made for centos and ubuntu distros.

Reply:
Comment content looks good, but could we do a manual line wrap in there so it's a bit easier to read?

Reply:
Done.